gqlalchemy/connection.py

```diff
@@ -107,21 +107,37 @@ def _create_connection(self) -> Connection:
 def _convert_memgraph_value(value: Any) -> Any:
     """Converts Memgraph objects to custom Node/Relationship objects"""
     if isinstance(value, mgclient.Relationship):
-        return Relationship(
-            rel_id=value.id,
-            rel_type=value.type,
-            start_node=value.start_id,
-            end_node=value.end_id,
-            properties=value.properties,
+        return Relationship.parse_obj(
+            {
+                "type": value.type,
+                "_id": value.id,
+                "_relationship_id": value.id,
+                "_relationship_type": value.type,
+                "_start_node_id": value.start_id,
+                "_end_node_id": value.end_id,
+                "_properties": value.properties,
+                **value.properties,
+            }
         )
 
     if isinstance(value, mgclient.Node):
-        return Node(node_id=value.id, labels=value.labels, properties=value.properties)
+        return Node.parse_obj(
+            {
+                "type": "".join(value.labels),
+                "_id": value.id,
+                "_node_id": value.id,
```

Comment:
I commented on this in the other thread. We can remove the `_id` and `_properties` fields. But I think there is value in keeping both. This is because sometimes you would like to keep objects in a hash map/dictionary and you can do so easily when there's a unified way of accessing the `_id` property.

I'm in favour of keeping just the `_id` property, but in this case I think there is value in being explicit. The issue arises from the following fields: `_node_labels` and `_relationship_type`. I think those will be misleading if we were to remove the `_node` and `_relationship` prefix. This is why I think there is a benefit in keeping both the `_id` property and the `_node_id` and `_relationship_id` properties.

Reply:
`_node_id` and `_relationship_id` have been removed, the `_properties` field is now a view.